lib/workers/repository/reconfigure/index.ts

```diff
@@ -48,14 +69,17 @@ export async function validateReconfigureBranch(
     return;
   }
 
-  const validationStatus = await platform.getBranchStatusCheck(
-    branchName,
-    'renovate/config-validation',
-  );
-  // if old status check is present skip validation
-  if (is.nonEmptyString(validationStatus)) {
-    logger.debug('Skipping validation check as status check already exists');
-    return;
+  if (context) {
+    const validationStatus = await platform.getBranchStatusCheck(
+      branchName,
+      context,
+    );
+
+    // if old status check is present skip validation
+    if (is.nonEmptyString(validationStatus)) {
+      logger.debug('Skipping validation check as status check already exists');
```

Comment:
```suggestion
      logger.debug('Skipping validation check because status check already exists.');
```

It think `because` is easier to understand, and we should end the sentence with a full stop.